stellar-contract-env-host/src/host.rs

```diff
@@ -751,18 +768,33 @@ impl Host {
         Ok(id_obj)
     }
 
+    fn try_call_vtable(
+        &self,
+        id: &Hash,
+        func: &Symbol,
+        args: &[RawVal],
+    ) -> Option<Result<RawVal, HostError>> {
+        #[cfg(feature = "testutils")]
+        if let Some(vtable) = self.0.vtables.borrow().get(&id) {
+            if let Some(f) = vtable.0.get(&func) {
+                let mut frame_guard = self.push_test_frame(id.clone());
+                let res = Some(Ok(f.0.call(args)));
+                frame_guard.commit();
+                res
+            } else {
+                Some(Err(HostError::General("function not in vtable")))
+            }
+        } else {
+            None
+        }
+
+        #[cfg(not(feature = "testutils"))]
+        None
```

Comment:
What's the intended future behavior of this fn when not in test mode, it seems odd to always return None.

Reply:
I reworked this to avoid this quirk.

Reply:
Is a commit missed from being pushed? This code is still present in the PR.

Reply:
I no longer see this code. Did you refresh?